### ISSUE TITLE
Chant search page filters have abbreviations only

### ISSUE THREAD

**Issue by dchiller** [DDMAL/CantusDB#1591]
The "Genre" and "Service" filter dropdowns on the [chant search page](https://cantusdatabase.org/chant-search/) display only abbreviations for those fields. This means that unless you are a CantusDB power user, you have to go to the separate pages that list the translation of these abbreviations to their names to filter effectively.

For example, this screenshot shows the possible values of "Genre" filter on the [chant search page](https://cantusdatabase.org/chant-search/):
<img width="800" alt="image" src="https://github.com/user-attachments/assets/b73182ff-8619-4352-9a4e-2281257442e8">

Was there a reason for this? I can't immediately find an issue that would suggest why this was implemented with the abbreviation only.

**Comment by dchiller** [comment]
@annamorphism @lucasmarchd01 thoughts or memories of why this was?

**Comment by lucasmarchd01** [comment]
#512, #699, #703 are responsible for the discussion around this issue.

I'm not entirely sure the reason we switched to using the abbreviations was justified. It seems like part of the reason was because the double square brackets looked strange. Instead maybe we can just come up with a better string representation for these models.

**Comment by dchiller** [comment]
thanks @lucasmarchd01 -- that makes sense. I guess it is good to have the abbreviation in the list so then when the filtered results show up in the table with the abbreviation, it's clear why the source is in the results. 

I'm wondering why we have some genre names with brackets. It looks like those are mostly deprecated genres? It looks like there are some chants with the `[?]` genre and only one with the `[M]` genre. None of the other bracketed genres look like they have chants...

I wonder if the [one chant with `[M]`](https://cantusdatabase.org/chant-search/?op=contains&keyword=&office=&genre=167&cantus_id=&mode=&feast=&position=&melodies=) should be reclassified? What does @annamorphism think about that?

And then I don't see why the `[?]` for unknown couldn't be renamed to just `?`?

**Comment by annamorphism** [comment]
Yes, that [M] should be reclassified--it's a little strange, actually, because that's a relatively recently added chant, so I wonder if the bracketed genres came back into use on NewCantus somehow after previously being deprecated. I also don't see why [?] couldn't be ? .

**Comment by kyrieb-ekat** [comment]
April25 meeting: yes still wanted! wanted on all pages where that dropdown is available.
